about/people.yml

```diff
@@ -1,177 +1,109 @@
-    team: Advanced Research Computing
+    team: Leadership
     subteam: Data & Visualization
-    title: Lead Data Scientist
+    title: Assistant Director of Research Software Engineering and Data Science
     github_username: mirestrepo
     brown_directory_uuid: cb8b0a49-ef4b-66e7-5ccc-6689493e8ace
     bio: Isabel grew up in Colombia and received her Ph.D from Brown in 2013 in Electrical
       Enginering with a focus in Computer Vision. She enjoys machine learning, reproducible
       science, reverse enginering, playing squash, the sea, and dancing with her wild
       children.
     image: isabel_main.jpg
-  - name: August Guang
+  - name: Linnea Wolfe
     type: Full Time
-    team: Advanced Research Computing
-    subteam: Computational Biology Core
-    title: Genomics Data Scientist
-    github_username: aguang
-    brown_directory_uuid: b3f2991e-b618-190f-881f-25fc21f82d02
-    bio: August obtained their PhD from Brown in 2017 in Applied Mathematics and Computational
-      Biology. Outside of work they organize against criminalization and the police
-      state. Both their research practices and organizing work are guided by the belief
-      that crisis and uncertainty are a gift and that we must be involved in shaping
-      how change happens.
-    image: august_main.jpg
-  - name: Andras Zsom
+    team: Leadership
+    title: Assistant Chief Information Officer, Infrastructure and Research Computing
+    github_username: ''
+    brown_directory_uuid: 3ddfe2bd-672a-0f09-03bc-cbb37339a434
+    bio: ''
+    image: linnea_main.jpg
+  - name: Mete Tunca
     type: Full Time
-    team: Advanced Research Computing
-    subteam: Institutional Data
-    title: Lead Data Scientist
-    github_username: azsom
-    brown_directory_uuid: 84734442-fa19-4e15-8b06-9f71351e9104
-    bio: Andras is passionate about using machine learning and predictive modeling for
-      good. He has a PhD in astrophysics and he was a postdoctoral researcher at MIT
-      before coming to Brown.
-    image: andras_main.jpg
-  - name: Ashley S. Lee
+    team: Leadership
+    title: Assistant Director of Research Systems and Services
+    github_username: mttunca
+    brown_directory_uuid: d0e7b3b6-f6b4-f28b-3bfd-08bccf05ac31
+    bio: ''
+    image: mete_main.jpg
+  - name: Paul Stey
     type: Full Time
-    team: Advanced Research Computing
-    subteam: Institutional Data
-    title: Senior Data Scientist
-    github_username: alee35
-    brown_directory_uuid: 380094e2-6f9d-47cf-91ce-e636876a58cf
-    bio: Ashley received training in Biology and Data Science at the University of Virginia
-      and dabbled in a variety of careers including Bike Salesman, Bus Driver, Bagel Slinger, and Chief
-      Fiduciary at Smoothie King. She enjoys food and beverage, bodies of water, jokes, and
-      caring for her beloved dog and plants.
-    image: ashley_main.jpg
+    team: Leadership
+    title: Associate Director of Research Systems and Services
```

Comment:
this title doesn't look right - I think @paulstey is an Assistant CIO of something or other

Reply:
Updated to `Director, Scientific Computing and Data Science, Adjunct Lecturer in International and Public Affairs
  Office of Information Technology`

Reply:
I'm pretty sure @paulstey got promoted to an assistant CIO - @paulstey what do you want here?

Reply:
Assistant CIO Research Software Engineering and Data Science

Reply:
Fixed !